python-backend/app/api/mines/mine_api_models.py

```diff
@@ -171,9 +171,19 @@ def format(self, value):
     'Variance', {
         'variance_id': fields.Integer,
```

Comment:
Really hate to not have noticed this before given all the discussion, but the integer PK shouldn't be exposed by our API and there should be a separate GUID that our API exposes. Integer PKs should be used internally by the DB for FKs, and the API can use them internally but not expose them to consumers. We had previously had discussion as a team led by Julian and settled on this pattern due to the efficiencies of integer PKs, but complications for database maintenance if we use an auto-incremented column as our global reference to an entity. I'd recommend merging this PR if it's blocking MD, then writing a migration script to add the GUID column and swap which is being exposed in a separate PR.

Reply:
Sure thing. I agree with that concern. I'd like to get this merged and open a new PR to make the guid changes